docs/sources/vendor/Broadcom/proxy.md

```diff
@@ -23,14 +23,14 @@ Broadcom products are inclusive of products formerly marketed under Symantec and
 
 | sourcetype     | notes                                                                                                   |
 |----------------|---------------------------------------------------------------------------------------------------------|
-| bluecoat:proxysg:access:kv        | Requires version TA 3.6                                                                                                    |
-| bluecoat:proxysg:syslog           | Requires version TA 3.6                                                                                                    |
+| bluecoat:proxysg:access:kv        | Requires version TA 3.8.1                                                                                                    |
+| bluecoat:proxysg:access:syslog           | Requires version TA 3.8.1                                                                                                    |
```

Comment:
Am I understood correctly that add-on maintainers broke backward compatibility (changed `sourcetype`) starting with 3.8.1 version ?

Can we talk with them to understand the reason?  Probably they will rollback this change if add-on not very popular :)

**If we need to break compatibility, probably good idea to provide some env var (`BLUECOAT_NEW_FORMAT=true`) that will determine we will assign old or new sourcetype for `bluecoat`. If will help us to prevent of breaking backward compatibility**

Reply:
that's a good question, 3.8.1 was released in September 2022 and I don't have access to older versions to check if that was changed on the TA side or there was a bug in SC4S

Reply:
especially that `bluecoat:proxysg:access:syslog` is more consistent with the rest than `bluecoat:proxysg:syslog`

Reply:
@mstopa-splunk you are right, it's impossible to check, because they deleted previous versions :)

Only 3.8.1 available on Splunkbase :)

Reply:
@ikheifets-splunk please see how this has been done previously:

https://splunk.github.io/splunk-connect-for-syslog/main/sources/vendor/Dell/sonicwall/
```
Note:[¶](https://splunk.github.io/splunk-connect-for-syslog/main/sources/vendor/Dell/sonicwall/#note)
The sourcetype has been changed in version 2.35.0 making it compliant with corresponding TA.
```

the practice seems to be updating the sourcetype and adding the note